cvat-ui/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config ./webpack.config.js",
-    "start": "webpack serve --env API_URL=http://localhost:7000 --config ./webpack.config.js --mode=development",
+    "start": "webpack serve --env API_URL=http://localhost:8080 --config ./webpack.config.js --mode=development",
```

Comment:
@PhoneHtet24 if we change port to the whole url would be changed for everyone

Reply:
right, bro. I will change it back.